sormas-backend/src/main/java/de/symeda/sormas/backend/region/GeoShapeProviderEjb.java

```diff
@@ -167,22 +164,23 @@ public GeoLatLon getCenterOfDistrict(DistrictReferenceDto district) {
 		}
 
 		Point polygonCenter = districtMultiPolygons.get(district).getCentroid();
-		return new GeoLatLon(polygonCenter.getX(), polygonCenter.getY());
+		return new GeoLatLon(polygonCenter.getY(), polygonCenter.getX());
 	}
 
 	@PostConstruct
 	private void loadData() {
 		String countryName = configFacade.getCountryName();
+		String WKT_String = configFacade.getGeocodingEPSG4326_WKTstring();
 		if (countryName.isEmpty()) {
 			logger.warn("Shape files couldn't be loaded, because no country name is defined in sormas.properties.");
 		} else {
-			loadRegionData(countryName);
-			loadDistrictData(countryName);
+			loadRegionData(countryName, WKT_String);
+			loadDistrictData(countryName, WKT_String);
 		}
 		buildCountryShape();
 	}
 
-	private void loadRegionData(String countryName) {
+	private void loadRegionData(String countryName, String WKT_String) {
```

Comment:
I think it's enough naming this parameter as `wkt` since the `_String` suffix seems redundant and it matches the naming of the parameter used by `CRS.parseWKT`